docs/src/rules/id-match.md

```diff
@@ -157,15 +157,15 @@ Examples of **incorrect** code for this rule with the default `"^[^_]+$", { "ign
 ```js
 /*eslint id-match: [2, "^[^_]+$", { "ignoreDestructuring": false }]*/
 
-var { category_id } = query;
+const { category_id } = query;
 
-var { category_id = 1 } = query;
+const { categoryid_Default = 1 } = query;
 
-var { category_id: category_id } = query;
+const { category_id: category_Id } = query;
```

Comment:
```suggestion
const { category_Id: category_Id } = query;
```

I think the point of this example was to have same names for property and variable.

Reply:
Sure, I renamed it to `category_ids` because `category_id` is already declared in the first line of this example.